src/main/kotlin/com/openlattice/admin/AdminController.kt

```diff
@@ -223,6 +224,22 @@ class AdminController : AdminApi, AuthorizingComponent {
         return jobService.getJobs(listOf(jobId))
     }
 
+    @Timed
+    @PatchMapping(
+            value = [JOBS + ID_PATH],
+            produces = [MediaType.APPLICATION_JSON_VALUE],
+            consumes = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    override fun updateJob(jobId: UUID, update: JobUpdate): Map<UUID, AbstractDistributedJob<*, *>> {
```

Comment:
Shouldn't there be `@PathVariable` annotations here?

```
fun updateJob(@PathVariable(ID) jobId: UUID, @PathVariable(UPDATE) update: JobUpdate)
```

Reply:
Yup, I was pushing up some incomplete local work so I could generate more incomplete local work >_<

Reply:
Also the second one should be ```@RequestBody``` not pathvariable.